wrappers/python/nntile/utils/constructors.py

```diff
@@ -0,0 +1,63 @@
+# @copyright (c) 2022-present Skolkovo Institute of Science and Technology
+#                              (Skoltech), Russia. All rights reserved.
+#                2023-present Artificial Intelligence Research Institute
+#                              (AIRI), Russia. All rights reserved.
+#
+# NNTile is software framework for fast training of big neural networks on
+# distributed-memory heterogeneous systems based on StarPU runtime system.
+#
+# @file wrappers/python/nntile/pipeline.py
+# TRaining pipeline of NNTile Python package
+#
+# @version 1.0.0
+
+import numpy as np
+
+from nntile.functions import fill_async
+from nntile.nntile_core.tensor import (
+    Tensor_bool,
+    Tensor_fp16,
+    Tensor_fp32,
+    Tensor_fp64,
+    Tensor_int64,
+    TensorTraits,
+)
+
+nnt2np_type_mapping = {Tensor_fp32: np.float32, Tensor_fp64: np.float64}
+
+np2nnt_type_mapping = {
+    np.dtypes.Float32DType: Tensor_fp32,
+    np.dtypes.Float64DType: Tensor_fp64,
+}
+
+
+def astensor(A: np.array, basetile_shape: tuple = None, mpi_distr=[0], next_tag=0):
+    A_traits = TensorTraits(A.shape, basetile_shape if basetile_shape else A.shape)
+
+    A_value = np2nnt_type_mapping[type(A.dtype)](A_traits, mpi_distr, next_tag)
+
+    A_value.from_array(A)
+    return A_value
+
+
+def asarray(tensor_nnt):
+    np_res = np.zeros(tensor_nnt.shape, order="F").astype(
+        nnt2np_type_mapping[type(tensor_nnt)]
+    )
+    tensor_nnt.to_array(np_res)
+    return np_res
+
+
+def zeros(shape, dtype=Tensor_fp32):
+    np_dtype = nnt2np_type_mapping[dtype]
+    return astensor(np.zeros(shape).astype(np_dtype))
```

Comment:
Please, do not do excessive conversion. You do `smth().astype()` almost everywhere.

```python
astensor(np.zeros(shape, type=np_dtype, order='F'))
```

Why not `full(shape, 0, dtype)`?

Reply:
Because it is not code, required performance and probably will be rewritten in near future. Fixed notation, looks prettier, thanks!